### PR TITLE
Docs: update to the latest

### PR DESCRIPTION
**Pull request by lxbsz**
Problem:
-------
There are many new feature or fixing has been added, but forgot to
update the docs.

Resolution:
---------
Update them to the upstream latest.

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -11,7 +11,7 @@ Install from Source:
 
 3. Run ./autogen.sh.
 
-4. Run ./configure.
+4. Run ./configure [--enable-tirpc=yea/no] [--disable-silent-rules] ....
```
Comment (pkalever) [inline INSTALL]: typo: s/yea/yes/

Comment (lxbsz) [reply INSTALL]: Fixed, thanks.

File: docs/gluster-block.8 [hunk 2]
```diff
@@ -102,8 +103,16 @@ show help message and exit.
...
+[\fBtimeout <seconds>\fR]
+it is the time in seconds that cli can wait for daemon to respond (default: 300)
+.PP
+
+.SS
+[\fB--json*\fR]
+used to request the output result in json format (default: plain text) and supported JSON formats: --json|--json-plain|--json-spaced|--json-pretty
+.PP
 
 .SH EXAMPLES
```
Comment (pkalever) [inline docs/gluster-block.8]: @lxbsz can you also correct:
To create a block device of size 1GiB, by preallocating storage with zero fill
.B # gluster-block create blockVol/sampleBlock prealloc full ${HOST} 1GiB

to

Some meaning ful msg here
.B # gluster-block create blockVol/sampleBlock prealloc no ${HOST} 1GiB

And also add an example for timeout option usage.

Thanks!

Comment (lxbsz) [reply docs/gluster-block.8]: Yeah, will fix it.
Thanks.

File: INSTALL [hunk 3]
```diff
@@ -11,7 +11,7 @@ Install from Source:
 
 3. Run ./autogen.sh.
 
-4. Run ./configure.
+4. Run ./configure [--enable-tirpc=yes/no] [--disable-silent-rules] ....
```
Comment (pkalever) [inline INSTALL]: @lxbsz also change the point 1, to reflect new targetcli recommended version.

Comment (lxbsz) [reply INSTALL]: Fixed it. Thanks.

File: docs/gluster-block.8 [hunk 4]
```diff
@@ -102,20 +103,34 @@ show help message and exit.
...
 .nf
 To create a block device of size 1GiB
 .B # gluster-block create blockVol/sampleBlock ${HOST} 1GiB
 
+To create a block device of size 1GiB with timeout 150 seconds
+.B # gluster-block timeout 150 create blockVol/sampleBlock ${HOST} 1GiB
```
Comment (pkalever) [inline docs/gluster-block.8]: @lxbsz 
The default is 300 sec, why would someone want to put timeout less than that :-)
Just an example usage, I understand, I will leave this up to you.

File: docs/gluster-block.8 [hunk 5]
```diff
@@ -102,20 +103,34 @@ show help message and exit.
...
+
 To create a block device of size 1GiB with auth enable
 .B # gluster-block create blockVol/sampleBlock auth enable ${HOST} 1GiB
 
 To create a block device of size 1GiB, by preallocating storage with zero fill
 .B # gluster-block create blockVol/sampleBlock prealloc full ${HOST} 1GiB
 
+To create a provision block device of size 1GiB
```
Comment (pkalever) [inline docs/gluster-block.8]: Should this be:
To create a thinly-provisioned block device of size 1GiB

Comment (lxbsz) [reply docs/gluster-block.8]: Fixed.
Thanks.

File: docs/gluster-block.8 (hunk 4 above)
Comment (pkalever) [reply docs/gluster-block.8]: @lxbsz I assume, you saw this and thought it is fine to go with 150 :-)

**Comment by pkalever** [comment]
Looks good to me. Thanks @lxbsz 
Reviewed-by: Prasanna Kumar Kalever \<prasanna.kalever@redhat.com\>

**Comment by lxbsz** [comment]
@pkalever 
The 150 just one example, will it matter here ? If so I will update it to 300 or larger one.

Thanks.

**Comment by pkalever** [comment]
> @pkalever
> The 150 just one example, will it matter here ? If so I will update it to 300 or larger one.

No, I'm okay. Just pointing it here, leave it to your choice.

Please update the tags and this is ready for merge. Thanks!

**Comment by lxbsz** [comment]
@pkalever 
Update with the tag and change 150 --> 300.
Thanks.

**Comment by pkalever** [comment]
Merged now, Thanks @lxbsz

**Pull request merged**